### PR TITLE
NOISSUE: support building docs using podman

### PR DESCRIPTION
**Pull request by dhellmann**
Version(s): all

Issue: none

Link to docs preview: n/a

QE review:
- [ ] QE has approved this change.

Additional information:

This PR updates the existing Dockerfile and adds a shell script to use the resulting image to run asciibinder instead of requiring all of the dependencies be installed on the local system. I'm using this and thought it might be useful to others.

/assign @kalexand-rh
/cc @ShaunaDiaz
/cc @fzdarsky

### CODE REVIEW

File: contributing_to_docs/tools_and_setup.adoc [hunk 1]
```diff
@@ -65,6 +65,7 @@ install the software and tools you will use to create the content. All OpenShift
 documentation is created in AsciiDoc, and is processed with https://github.com/redhataccess/ascii_binder[AsciiBinder],
 which is an http://asciidoctor.org/[AsciiDoctor]-based docs management system.
 
+NOTE: If you do not want to install the tools directly onto your system, you can use `build.sh` to run the tools in a container via podman.
```
Comment (kalexand-rh) [inline contributing_to_docs/tools_and_setup.adoc:68]: Are there other requirements besides having podman installed? I installed podman and started the container, as it suggested, but can't get this to work.

Comment (dhellmann) [reply contributing_to_docs/tools_and_setup.adoc:68]: You should need anything else, but it's possible I've made some assumptions. What OS are you on (it probably only works on Fedora, CentOS, or RHEL)? Can you share the output you're seeing? (gist, slack, whatever's easy)

Comment (kalexand-rh) [reply contributing_to_docs/tools_and_setup.adoc:68]: I'm on mac. I assumed that because I could install podman that this would work. :)

Comment (dhellmann) [reply contributing_to_docs/tools_and_setup.adoc]: The latest update worked somewhat on my M1 Mac. There were some warnings about include files and images that binder couldn't read but that are definitely present. I'm testing on x86 now.

Comment (dhellmann) [reply contributing_to_docs/tools_and_setup.adoc]: I saw similar errors on x86, so I updated the disclaimer to suggest that it does not work on macOS, yet.

**Comment by dhellmann** [comment]
Now that quay.io/openshift-cs/asciibinder is being published, I don't think this PR is needed any longer.

**Pull request closed without merging**